Formula/blocky.rb

```diff
@@ -0,0 +1,76 @@
+class Blocky < Formula
+  desc "Fast and lightweight DNS proxy as ad-blocker with many features"
+  homepage "https://0xerr0r.github.io/blocky"
+  url "https://github.com/0xerr0r/blocky/archive/refs/tags/v0.20.tar.gz"
+  sha256 "aae5346e9c1ce4b326b9e578939aa26ddca39338d79d0ddb3eb079ae7a949e87"
+  license "Apache-2.0"
+  head "https://github.com/0xerr0r/blocky.git", branch: "development"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X github.com/0xERR0R/blocky/util.Version=#{version}
+      -X github.com/0xERR0R/blocky/util.BuildTime=#{time.iso8601}
+    ]
+    system "go", "build", *std_go_args(ldflags: ldflags, output: bin/"blocky")
```

Comment:
if you do sbin symlink, then `output` would be redundant, by default we install binary in the bin folder.

```suggestion
    system "go", "build", *std_go_args(ldflags: ldflags)
```

Reply:
also you can generate completion for client as well, let me add some code suggestion for it.